proxy/server.py

```diff
@@ -344,6 +346,11 @@ def do_GET(self):
         except:
             log.error("Socket broken sending response [doGET]")
 
+def sigTermHandle(signum, frame):
+    sys.exit()
```

Comment:
My recommendation/preference would be to use `raise SystemExit` per explanation [here](https://github.com/jasonacox/Powerwall-Dashboard/issues/353?notification_referrer_id=NT_kwDOBnsFf7Q3NzcxNzQzMTA2OjEwODcyNTYzMQ#issuecomment-1728709935)